### PR TITLE
{numlib}[GCCcore/9.3.0] Triangle v1.6

### PR DESCRIPTION
**Pull request by SimonPinches**

### CODE REVIEW

File: easybuild/easyconfigs/t/Triangle/Triangle-1.6-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+easyblock = 'MakeCp'
+
+name = 'Triangle'
+version = '1.6'
+
+homepage = 'http://www.cs.cmu.edu/~quake/triangle.html'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/t/Triangle/Triangle-1.6-GCCcore-9.3.0.eb:6]: please switch to https

File: easybuild/easyconfigs/t/Triangle/Triangle-1.6-GCCcore-9.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+description = """Triangle generates exact Delaunay triangulations, constrained Delaunay triangulations,
+ conforming Delaunay triangulations, Voronoi diagrams, and high-quality triangular meshes.
+ The latter can be generated with no small or large angles,
+ and are thus suitable for finite element analysis."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.netlib.org/voronoi/']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/t/Triangle/Triangle-1.6-GCCcore-9.3.0.eb:15]: same as above

**Review by migueldiascosta — DISMISSED** [review]

File: easybuild/easyconfigs/t/Triangle/Triangle-1.6-GCCcore-9.3.0.eb (hunk 1 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/t/Triangle/Triangle-1.6-GCCcore-9.3.0.eb]: Done

File: easybuild/easyconfigs/t/Triangle/Triangle-1.6-GCCcore-9.3.0.eb (hunk 2 above)
Comment (SimonPinches) [reply easybuild/easyconfigs/t/Triangle/Triangle-1.6-GCCcore-9.3.0.eb]: Done.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
himem06.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Platinum 8160 CPU @ 2.10GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/jfgrimm/6fc716de4afa3d2044c96470a39efee5 for a full test report.

**Review by jfgrimm — APPROVED** [review]
lgtm

**Pull request merged**